controls/def.cf

```diff
@@ -349,7 +349,8 @@ bundle common inventory_control
 # are disabled.
 {
   vars:
-      "lldpctl_exec" string => "/usr/bin/lldpctl";
```

Comment:
I think these should be moved inside the bundle they are used in. Set to the default value if there is no def.lldpctl_json defined.

What do you think?
At least lldpctl_json needs to get ifvarclass => not(isvariable(def.lldpctl_json)) so that users can simply define lldpctl_json from def.json to use something like your special perl script.